server/log.js

```diff
@@ -18,17 +18,40 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-var winston = require('winston');
-require('winston-daily-rotate-file');
-
-/* Initialize rotating logging. */
-exports.init = function(callback) {
-    winston.add(winston.transports.DailyRotateFile, {
-        filename: 'log/quizzard',
-        datePattern: 'yyyy-MM-dd-',
-        prepend: true
-    });
-    callback();
+var fs = require('fs');
+var common = require('./common.js');
+var dateStamp;
+var logger;
+
+// print a normal message
+exports.log = function (text) {
+    init();
+
+    var txt = common.formatString('[{0}] log: {1}', [common.getDate(), text]);
+    console.log(txt);
+}
+
+// print an error message
+exports.error = function (text) {
+    init();
+
+    var txt = common.formatString('[{0}] error: {1}', [common.getDate(), text]);
+    console.error(txt);
```

Comment:
do we really need that extra var? can we just do:
``` javascript
console.error(common.formatString('[{0}] error: {1}', [common.getDate(), text]));
```
This also applies to the `console.log()` above

Reply:
Not now. But I am planing to add more information to the logs, such as the userId and parts of the me object, which will make this line much longer.